src/disk-images/github-runner-linux/config.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
```

Comment:
To keep things consistent should this also include a `# Copyright © 2023 SUSE LLC` header?

Reply:
Added.